setup.cfg

```diff
@@ -16,33 +16,28 @@ version = 2022.11.19
 #            versions that are no longer supported. This may break installing
 #            older versions of packages that depend on this meta-package!
 [options]
-python_requires = >=3.5
+python_requires = >=3.7
 install_requires =
     # NOTE: Any platform-related assumptions made here are encoded in the
     #       tests for this package.
 
     # AIX requires fixes contained in numpy 1.16
-    numpy==1.16.0; python_version=='3.5' and platform_system=='AIX'
-    numpy==1.16.0; python_version=='3.6' and platform_system=='AIX'
-    numpy==1.16.0; python_version=='3.7' and platform_system=='AIX'
+    numpy==1.16.0; python_version=='3.7' and platform_system=='AIX' and platform_machine!='loongarch64' and platform_python_implementation != 'PyPy'
 
     # IBM i requires fixes contained in numpy 1.23.3. Only Python 3.6 and 3.9
     # are supported on IBM i system, so we're ignoring other Python versions here
     # for simplicity (see gh-66).
-    numpy==1.23.3; python_version=='3.9' and platform_system=='OS400'
+    numpy==1.23.3; python_version=='3.9' and platform_system=='OS400' and platform_machine!='loongarch64'
 
     # numpy 1.19 was the first minor release to provide aarch64 wheels, but
-    # wheels require fixes contained in numpy 1.19.2 (For Python 3.5: support
-    # was dropped in 1.19 so numpy 1.18.5 can be built from source instead).
-    numpy==1.18.5; python_version=='3.5' and platform_machine=='aarch64' and platform_python_implementation != 'PyPy'
-    numpy==1.19.2; python_version=='3.6' and platform_machine=='aarch64' and platform_python_implementation != 'PyPy'
-    numpy==1.19.2; python_version=='3.7' and platform_machine=='aarch64' and platform_python_implementation != 'PyPy'
-    numpy==1.19.2; python_version=='3.8' and platform_machine=='aarch64' and platform_python_implementation != 'PyPy'
+    # wheels require fixes contained in numpy 1.19.2
+    numpy==1.19.2; python_version=='3.7' and platform_machine in 'aarch64|ppc64le' and platform_system!='AIX' and platform_python_implementation != 'PyPy'
```

Comment:
Is the comment above also reflective of the situation on `ppc64le`?

Reply:
If we're talking compatibility only, ppc was added (with tests) in conda-forge for 1.16 and passed.

I guess realistically aarch/ppc should have _better_ compatibility than s390x, where we're only claiming 1.17.5.

That said, I just wanted to add it at all, and aarch seemed like a good reference. I'm happy to put whatever between 1.16 & 1.19 as a baseline.

Reply:
`aarch64` is set to 1.19.2 here because 1.19 is the first minor version that added wheels for that platform (and the `.2` because of a bug fix): https://pypi.org/project/numpy/1.19.2/#files.

There are no wheels for `ppc64le`, so I don't know why this requires special-casing here, or a change at all.

> That said, I just wanted to add it at all,

It is added, _every_ platform is. It's covered by the default requirements unless it's special-cased for a specific reason (and if so, it's then excluded from the relevant default requirement). So I suggest you revert this change.

Reply:
Those were added each time in response to concrete bug fixes that landed in a particular NumPy release. The rule is: every platform is covered by the default requirements, unless there's a reason to bump up the min version. I'd like to stick to that, since it has worked well enough, and any other change has a nontrivial change of breaking something.

Reply:
> The rule is: every platform is covered by the default requirements, unless there's a reason to bump up the min version.

Done this for linux-ppc & win-arm64 now